systems/llnl-sierra/system.py

```diff
@@ -5,11 +5,37 @@
 
 import pathlib
 
-from benchpark.directives import variant
+from benchpark.directives import variant, maintainers
 from benchpark.system import System
+from benchpark.paths import hardware_descriptions
 
 
 class LlnlSierra(System):
+
+    id_to_resources = {
+        "lassen": {
+            "sys_cores_per_node": 44,
+            "sys_gpus_per_node": 4,
+            "hardware_key": str(hardware_descriptions)
+            + "/IBM-power9-V100-Infiniband/hardware_description.yaml",
+        },
+        "sierra": {
+            "sys_cores_per_node": 44,
+            "sys_gpus_per_node": 4,
+            "hardware_key": str(hardware_descriptions)
+            + "/IBM-power9-V100-Infiniband/hardware_description.yaml",
+        },
+    }
+
+    system_site = "llnl"
+
+    variant(
```

Comment:
Lassen and Sierra are identical, I am not convinced these should be considered variants.

Reply:
This is fixed in [c8e9fcf](https://github.com/LLNL/benchpark/pull/587/commits/c8e9fcfe4a1aa7525418ba797baeed6cb4656366)